adam/experiment/log_experiment.py

```diff
@@ -356,7 +481,7 @@ def curriculum_from_params(
     ]
 
     num_samples = params.optional_positive_integer("num_samples")
-    num_noise_objects = params.optional_positive_integer("num_noise_objects")
+    num_noise_objects = params.optional_integer("num_noise_objects")
```

Comment:
Why the change? Do we want to allow zero as an explicit value?

(It's odd, but [apparently](https://github.com/isi-vista/vistautils/blob/ffdc70e900715d7befd5c0e349371f6f58d180cc/vistautils/parameters.py#L564) vistautils doesn't support specifying a range on optional integers as opposed to required ones, otherwise we could use that.)

Reply:
@LichMaster98 I agree, and I think that solution makes sense.